src/resources/js/tec-update-6.0.0-notice.js

```diff
@@ -0,0 +1,22 @@
+( function ( wp ) {
+
+    wp.data.dispatch( 'core/notices' )
+        .createNotice(
+            'info', 
+            '<b>' +  data.title + '</b><p>' + data.description + '</p>', 
+            { 
+                __unstableHTML: true, 
+                isDismissible: true, 
+                actions: [ 
+                    { 
+                        url: data.upgrade_link, 
+                        label: 'Start storage migration' 
+                    }, 
+                    { 
+                        url: data.learn_link, 
+                        label: 'Learn more' 
```

Comment:
This should be translatable. [See more](https://wp.zacgordon.com/2018/01/09/how-to-use-wp-i18n-__-to-allow-for-translation-of-your-gutenberg-block-text/)

Reply:
done